docs/src/content/mapping/lodash/some.md

```diff
@@ -0,0 +1,17 @@
+---
+category: Collection
+---
+
+_Not provided by Remeda._
+
+Use the native JS [`Array.prototype.some`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) instead.
+
+_**@see** ["You don't need Lodash"](https://you-dont-need.github.io/You-Dont-Need-Lodash-Underscore/#/?id=_some)_
```

Comment:
**question:** does this work? what does this look like?

Reply:
![image](https://github.com/remeda/remeda/assets/4711558/2c0106fb-26f3-4499-98ec-2e025e2aaaf1)

Reply:
hm...... it's not great, but it's fine for now